### DISCUSSION TITLE
Prevent ability to click circles after fail

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17222] [Ideas]
It feel like it ruins the gameplay, I recommend disabling hitsounds after fail

https://user-images.githubusercontent.com/40531871/157955823-0c20cac2-aacd-425c-8d09-56581d49e614.mp4

**Comment by integerrr** [comment]
#12166, ppy did say he will reconsider if someone else brings it up again so...

**Reply by AuracleTech** [reply]
Oh well

**Comment by frenzibyte** [comment]
Since the fail animation has changed to scale the entire gameplay screen away from the player, I think it sort of makes sense to remove the ability of interacting with it. Curious to hear @peppy's take on this, however.

**Reply by AuracleTech** [reply]
I feel like it's more of a question of satisfaction.
Hitsounds is one of the most satisfying thing in osu, so being able to hear hitsounds at a moment where you lost is kinda... off-putting.
It feels uncanny valley
